addons/auth_signup/models/res_config_settings.py

```diff
@@ -16,7 +16,7 @@ class ResConfigSettings(models.TransientModel):
             ('b2c', 'Free sign up'),
         ],
         string='Customer Account',
-        default='b2b',
+        default='b2c',
```

Comment:
Why are you changing this default?

Reply:
For consistency with
https://github.com/odoo/odoo/blob/5ec010de2f57f25935680a319b8c0fcdbb18b3ab/addons/auth_signup/data/ir_config_parameter_data.xml#L5

Currently, the setting appears to be `b2c` in the config, and users have to (and do) make dirty and 'save' the setting. Also, we felt this way was functionally right (vs changing the data to `b2b`).

Reply:
> Also, we felt this way was functionally right (vs changing the data to b2b).

Not the subject of this task but I still feel that it's a bit unexpected to allow signup by default, especially when you have some unrelated module like account or project.

Reply:
Yeah I agree but this is more a "business" talk to have with people that don't want their customers to configure something...